lesson_07/dags/process_sales.py

```diff
@@ -0,0 +1,107 @@
+import os
+import json
+from datetime import datetime, timedelta
+from typing import Any
+
+from airflow import DAG
+from airflow.models import TaskInstance
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+from airflow.providers.http.hooks.http import HttpHook
+from airflow.utils.context import Context
+from airflow.utils.trigger_rule import TriggerRule
+
+# Project path
+PROJECT_PATH = "/Users/r/PycharmProjects/Data-Engineering/lesson_07"
```

Comment:
the local path does not need to be stored in the code, 
for example we can add this in Airflow Variable